### DISCUSSION TITLE
Incorrect calling enqueue

### DISCUSSION THREAD

**Discussion by RobertCrx** [bootscore/bootscore#549] [Q&A] [answered]
I'm getting this warning on all my pages and I suppose this comes from ```/woocommerce/wc-functions.php``` 
and here ```wp_enqueue_script('wc-cart-fragments');```

Warning: ```Function wp_enqueue_script was called incorrectly. Scripts and styles should not be linked or registered before the wp_enqueue_scripts, admin_enqueue_scripts, or login_enqueue_scripts actions run. This notification was triggered by handling wc-cart-fragments. (This message was added in version 3.3.0.)```

How can I wrap this function inside add_action( 'init', '', 10 ) in my child theme ?
Thank you.

**Comment by crftwrk** [comment, marked as answer]
Which theme version are you using? This was fixed in v5.3.1 https://github.com/bootscore/bootscore/releases.

Please update to latest version.

Solved?

**Reply by RobertCrx** [reply]
I will update the theme , thank you!
